blockly/frontend/index.html

```diff
@@ -1,35 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Blockly Dungeon</title>
-    <link rel="hsbi-icon" href="/favicon.ico" type="image/x-icon" />
-  </head>
-  <body>
-    <header>
-      <h1 id="page-title">Blockly Dungeon</h1>
-      <img id="cat-logo" src="/cat_logo.png" alt="cat-logo" />
-    </header>
-    <div id="pageContainer">
-      <div id="blockly"></div>
-      <div id="outputPanel">
-        <pre id="generatedCode"><code></code></pre>
-        <div id="response">
-          <div id="responseStatus">HTTP response status:</div>
-          <div id="characterPosition">Character position:</div>
-        </div>
-        <div id="config_div">
-          <label for="delay">Verzögerung (in Sek.)</label>
-          <input type="text" inputmode="numeric" id="delay" name="delay" min="1" max="5">
-        </div>
-        <div id="buttons">
-          <button type="button" id="startBtn">Start</button>
-          <button type="button" id="stepBtn">Step</button>
-          <button type="button" id="resetBtn">Reset</button>
-        </div>
-      </div>
-    </div>
-    <script type="module" src="/src/index.ts"></script>
-  </body>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Blockly Dungeon</title>
+  <link rel="hsbi-icon" href="/favicon.ico" type="image/x-icon" />
+  <style>
+      #buttons {
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        background-color: white;
+        padding: 10px;
+        text-align: center;
+        box-shadow: 0 -2px 5px rgba(0, 0, 0, 0.1);
+      }
+    </style>
```

Comment:
Styles bitte in die css Datei

Reply:
Die css hat schon folgende Def: 
```css
#buttons {
  height: 30%; /* <- Hier das ist aktuell noch ein Problem, das muss weg */
  display: flex;
  justify-content: center;
  align-items: center;
}
```

Am ende sollte es da so aussehen:
```css
#buttons {
  display: flex;
  justify-content: center;
  align-items: center;
  position: fixed;
  bottom: 15px;
  width:100%;
  left: 10%;
}
```

Ich habe mal ein paar Sache getestet vlt sieht das ja auch bei dir gut aus.

Reply:
passt doch.

done

Hab die Button Reihenfolge noch geändert

Step Start Reset 

dann wird der wichtige Start Button quasi nie hinter anderen Elementen versteckt (wenn man das Fenster sehr klein macht).